aws/cloudformation/moz-single.json

```diff
@@ -365,6 +365,14 @@
             }
           },
           {
+            "Name" : { "Fn::Join" : [ "", ["321done-untrusted-", {"Ref" : "Subdomain"}, ".", {"Ref" : "HostedZone"}, "."]]},
```

Comment:
In the hostname, and later in the url path /123done-untrusted, I would drop the `-untrusted` part. Just seems too fugly.